### PR TITLE
XDR mirroring bug fix

### PR DESCRIPTION
**Pull request by dansterenson**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/41457

## Description
In Demisto version 6.1.0 and above, the get_remote_data_command will only be automatically executed on changed incidents
returned from get_modified_remote_data_command so we want to perform extra-data request on those incidents and there is no need to check again if_incident_was_modified_in_xdr.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by dansterenson** [comment]
@bakatzir can you please go over the release notes?

File: Packs/CortexXDR/ReleaseNotes/4_1_11.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Fixed an issue where the integration did not handle mirroring properly.
```
Comment (bakatzir) [inline Packs/CortexXDR/ReleaseNotes/4_1_11.md:4]: can you specify in which cases? so the customer will know if it is relevant for him to upgrade the pack.
```suggestion
- Fixed an issue where the integration did not handle mirroring properly.
```

Comment (dansterenson) [reply Packs/CortexXDR/ReleaseNotes/4_1_11.md:4]: There was a problem with the incidents update time, I am not sure we should get too specific, what do you think?

Comment (bakatzir) [reply Packs/CortexXDR/ReleaseNotes/4_1_11.md:4]: usually you want to be specific.

**Pull request merged**